js/components/seismic-eruptions-map.js

```diff
@@ -93,7 +100,11 @@ export default class SeismicEruptionsMap extends PureComponent {
       // TOOD: remove, it can be useful to tweak position of plate arrows.
       console.log('lat:', e.latlng.lat, 'lng:', e.latlng.lng)
     })
-
+    if (!config.sizeToFitBounds) {
+      // if we are not using bounds, and instead using a center point, force a refresh of the viewport to
+      // get the earthquakes to render on first load
+      setTimeout(() => this.handleMapViewportChanged(), 400)
```

Comment:
This seems a bit random. Why timeout and why 400ms?

Reply:
copy/paste from elsewhere - good catch!